### PR TITLE
Updated README so that it contains install instructions

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Moved install instructions to README. We can then remove the wiki page when merged

### CODE REVIEW

**Review by rhfogh — APPROVED** [review]
Looks fine.

File: README.md [hunk 1]
```diff
@@ -2,32 +2,40 @@
...
 [![codecov](https://codecov.io/gh/mxcube/mxcube3/branch/master/graph/badge.svg)](https://codecov.io/gh/mxcube/mxcube3)
 
-
 <p align="center"><img src="http://mxcube.github.io/mxcube/img/mxcube_logo20.png" width="125"/></p>
 
 # MXCuBE-Web
+
+
+MXCuBE-Web is the latest generation of the data acquisition software MXCuBE (Macromolecular Xtallography Customized Beamline Environment).
+
```
Comment (beteva) [inline README.md]: repetition

File: README.md [hunk 2]
```diff
@@ -55,3 +61,152 @@ Oscarsson, M. et al. 2019. “MXCuBE2: The Dawn of MXCuBE Collaboration.” Jour
...
+# or (to pass a different name)
+conda env create -f conda-environment.yml --name another_name
+```
+
+### 4. Activate the environment
+
+```
+conda activate mxcubeweb
+```
+
```
Comment (beteva) [inline README.md]: Do we oblige to use conda or we strongly recommend it, but leave it optional?

Comment (marcus-oscarsson) [reply README.md]: Id say that we recommend it but one is of course free to choose another way of installing things

Comment (beteva) [reply README.md]: In this case, maybe we say something like - We use conda for a convenience and to be independent from other projects libraries and python version. This, though, is not compulsory and one can use any other tool or configuration for installing the application and the necessary third party libraries.

**Review by meguiraun — APPROVED** [review]

**Pull request merged**